src/test/java/com/myhome/services/integration/HouseMemberDocumentServiceIntegrationTest.java

```diff
@@ -0,0 +1,270 @@
+import com.myhome.domain.HouseMemberDocument;
+import com.myhome.repositories.HouseMemberDocumentRepository;
+import com.myhome.repositories.HouseMemberRepository;
+import com.myhome.repositories.UserRepository;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.mock.web.MockMultipartFile;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@AutoConfigureMockMvc
+@TestPropertySource(properties = {
+    "files.maxSizeKBytes=1",
+    "files.compressionBorderSizeKBytes=99",
+    "files.compressedImageQuality=0.99"
+})
+public class HouseMemberDocumentServiceIntegrationTest extends ControllerIntegrationTestBase {
+
+  private static final String MEMBER_ID = "default-member-id-for-testing";
+  private static final String TEST_DOCUMENT_NAME = "test-document";
+  private static final String testUserName = "Test User";
+  private static final String testUserEmail = "testuser@myhome.com";
+  private static final String testUserPassword = "testpassword";
+  @Autowired
+  private HouseMemberDocumentRepository houseMemberDocumentRepository;
+  @Autowired
+  private HouseMemberRepository houseMemberRepository;
+  @Autowired
+  private UserRepository userRepository;
+  @Autowired
+  private MockMvc mockMvc;
+
+  @BeforeEach
+  void setUp() {
+    if(!houseMemberRepository.findByMemberId(MEMBER_ID).isPresent()) {
+      HouseMember member = houseMemberRepository.save(new HouseMember(MEMBER_ID, null, "test-member-name", null));
+      member.setMemberId(MEMBER_ID);
+      houseMemberRepository.save(member);
+    }
+    authDefaultUser();
+  }
+
+  @AfterEach()
+  void cleanHouseMemberDocument() {
+    HouseMember member = houseMemberRepository.findByMemberId(MEMBER_ID).get();
+    member.setHouseMemberDocument(null);
+    houseMemberRepository.save(member);
+  }
+
+  @Test
+  void getHouseMemberDocumentSuccess() throws Exception {
+    // given
+    addDefaultHouseMemberDocument();
+    // when
```

Comment:
Please leave one blank line between given-when and when-then.
Applicable for all occurences